RecoHI/HiTracking/python/RecoHiTracker_EventContent_cff.py

```diff
@@ -47,6 +49,7 @@
 RecoHiTrackerAOD = cms.PSet(
     outputCommands = cms.untracked.vstring('keep recoTracks_hiGeneralTracks_*_*',
                                            'keep recoTracks_hiGeneralAndPixelTracks_*_*',
-                                           'keep recoVertexs_hiSelectedVertex_*_*'		
+                                           'keep recoVertexs_hiSelectedVertex_*_*',		
+                                           'keep recoVertexs_hiSelectedPixelVertex_*_*'		
```

Comment:
This is a good point, comparisons between the two collections can probably be done with a small sample of RECO.  I have removed it from the AOD, but left it in RECO.